.github/workflows/go.yml

```diff
@@ -28,7 +28,6 @@ jobs:
     env:
       GOPATH: ${{ github.workspace }}
       WORKING_DIR: ${{ github.workspace }}/src/github.com/percona/pmm-agent
-      GO111MODULE: off #TODO: remove once we move to go modules https://jira.percona.com/browse/PMM-7106
```

Comment:
Let's add go modules cache to CI
https://github.com/percona-platform/platform/blob/77210c15d1db136a396d9f382b97c170877410d0/.github/workflows/ci.yml#L66-L72

and cache cleaner 
https://github.com/percona-platform/platform/blob/main/.github/workflows/clean.yml

Reply:
added